contracts/token/ArcxRedemption.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.4;
+
+import {Ownable} from "../lib/Ownable.sol";
+import {IERC20} from "../token/IERC20.sol";
+import {SafeERC20} from "../lib/SafeERC20.sol";
+
+
+contract ArcxRedemption is Ownable {
+
+    using SafeERC20 for IERC20;
+
+    IERC20 public usdcToken;
+
+    IERC20 public arcxToken;
+
+    uint256 public exchangeRate;
+
+    uint256 public cutoffDate;
```

Comment:
Even the cutoff date could be initialized right away here, leaving the constructor empty :)